pkg/proxy/iptables/proxier.go

```diff
@@ -1578,20 +1578,6 @@ func (proxier *Proxier) syncProxyRules() {
 	err = proxier.iptables.RestoreAll(proxier.iptablesData.Bytes(), utiliptables.NoFlushTables, utiliptables.RestoreCounters)
 	if err != nil {
 		glog.Errorf("Failed to execute iptables-restore: %v", err)
-		// ~rough approximation, assume ~100 chars per line
-		// we log first 1000 bytes, but full list at higher levels
-		rules := proxier.iptablesData.Bytes()
-		if len(rules) > 1000 {
-			abridgedRules := rules[:1000]
-			if glog.V(4) {
-				glog.V(4).Infof("Rules:\n%s", rules)
```

Comment:
I would expect you to preserve this at the higher debug level.  How else is someone going to debug it?

Reply:
https://github.com/kubernetes/kubernetes/pull/54700/files#diff-d51765b83fe795b469e8a86276b12dc9L1577

Reply:
ok, lgtm then

Reply:
@smarterclayton a few lines above displays the iptable rules at v5. You can get the info by logging at v5